app/views/main.html

```diff
@@ -4,45 +4,96 @@
             <div class="panel-heading">
                 <div class="col-md-offset-4">
                     <h2 class="nav-start">{{mainCtrl.message}}</h2>
-                    <input type="number" ng-model="mainCtrl.amount" aria-label="amount">
-                    <button ng-click="mainCtrl.wallet.pln = mainCtrl.amount">Nowa kwota startowa</button>
-                    <button ng-click="mainCtrl.wallet.$reset({})">Clear localStorage</button>
+                    <form name="formm" novalidate>
+                        <div>
+                            <input type="number" min="1" max="100000000" ng-model="mainCtrl.amount" ng-model-options="size" name="size" aria-label="amount"
+                                   integer/>
+                            <button ng-click="mainCtrl.buttonStart()">Nowa kwota startowa</button>
+                            <br/>
+                            <p ng-show="formm.size.$error.min || formm.size.$error.max">
+                                Podaj wartosc wieksza od 0 a mniejsza od 100 000 000!</p>
+                        </div>
+                    </form>
                 </div>
             </div>
         </div>
     </div>
 
-    <div class="col-md-12">
+    <div class="col-md-7">
         <div class="panel panel-default">
             <div class="panel-heading"><h2>Aktualny kurs walut i stan srodkow</h2></div>
             <div class="panel-body">
-
                 <table>
                     <tr>
                         <th>
                             Waluta
                         </th>
                         <th>
+                            Kod waluty
+                        </th>
+                        <th>
                             Kupno
                         </th>
                         <th>Sprzedaz</th>
-                        <th class="oneLine">Posiadana Kwota</th>
-                    </tr>
-                    <tr>
-                        <th>PLN</th>
-                        <th></th>
-                        <th></th>
-                        <th>{{mainCtrl.wallet.pln}}</th>
                     </tr>
                     <tr ng-repeat="currency in mainCtrl.arrayCurrency">
+                        <th>{{currency.currency}}</th>
                         <th>{{currency.code}}</th>
                         <th>{{currency.ask}}</th>
                         <th>{{currency.bid}}</th>
-                        <th class="oneLine">--</th>
                     </tr>
                 </table>
+            </div>
+        </div>
+    </div>
 
+    <div class="col-md-5">
+        <div class="panel panel-default">
+            <div class="panel-heading"><h2>Stan srodkow</h2></div>
+            <div class="panel-body">
+                <table>
```

Comment:
This table look werid. Try to create table in this way:

```
<table border="1" width="100%">
        <thead>
                <tr>
                        <th>Row 1, cell 1</th>
                        <th>Row 1, cell 2</th>
                        <th>Row 2, cell 3</th>
                </tr>
        </thead>
        <tbody>
                <tr>
                        <td>Row 2, cell 1</td>
                        <td>Row 2, cell 2</td>
                        <td>Row 2, cell 3</td>
                </tr>
        </tbody>
</table>
```
with `th` in `thead` and `tr` in `tbody`.

I think you should try to use here ng-repeat.